<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenGLModelDisplayerClass</class>
 <widget class="QMainWindow" name="OpenGLModelDisplayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>OpenGLModelDisplayer</string>
  </property>
  <widget class="QWidget" name="widget_central_">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QFrame" name="frame_gl_canvas_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_switch_rotate_mode_">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>400</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch rotate mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset_eye_">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>430</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset eye</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hint_">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>180</width>
      <height>150</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Press mouse : adjust angle
Scroll mouse : Change eyes distance</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_bar_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file_">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_obj_"/>
    <addaction name="action_load_texture"/>
   </widget>
   <widget class="QMenu" name="menu_mode_">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="action_view_basic_demo_"/>
    <addaction name="action_view_obj_"/>
    <addaction name="action_view_robot_"/>
   </widget>
   <addaction name="menu_file_"/>
   <addaction name="menu_mode_"/>
  </widget>
  <widget class="QToolBar" name="tool_bar_main_">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="status_bar_"/>
  <action name="action_open_obj_">
   <property name="text">
    <string>Open obj</string>
   </property>
  </action>
  <action name="action_view_obj_">
   <property name="text">
    <string>View obj</string>
   </property>
  </action>
  <action name="action_view_robot_">
   <property name="text">
    <string>View robot</string>
   </property>
  </action>
  <action name="action_view_basic_demo_">
   <property name="text">
    <string>View basic demo</string>
   </property>
  </action>
  <action name="action_load_texture">
   <property name="text">
    <string>Load texture</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="opengl_model_displayer.qrc"/>
 </resources>
 <connections/>
</ui>
